datacube_wps/processes/mangrovedrill.py

```diff
@@ -3,6 +3,27 @@
 
 from . import PolygonDrill, chart_dimensions, log_call
 
+# Mangrove cover product version 3 (ga_ls_mangrove_cover_cyear_3) does not contain crs,resolution in its products def
+# - https://cmi.ga.gov.au/data-products/dea/634/dea-mangrove-canopy-cover-landsat#processing = UTM, 30 m
+# - https://cmi.ga.gov.au/data-products/dea/191/dea-mangrove-canopy-cover-landsat-deprecated#processing = Albers, 25 m
+# Need to pass through load_hints (output_crs, resolution) to the geobox calculations
+# 1. test_api calls datacube_wps.processes.__init__.PolygonDrill.query_handler()
+#    - query_handler accepts a "parameters" arg that could be used
+# 2. query_handler calls self.input_data()
+#    - the parameters arg is not passed in here but could be
+# 3. input_data calls self.input.* = datacube.virtual.Product()
+#    - https://github.com/opendatacube/datacube-core/blob/develop/datacube/virtual/impl.py#L313
+#    - the Product() is created from datacube-wps-config.yaml by .impl.create_process(), which calls datacube.virtual.construct()
+# 4. bag = self.input.query(...)
+#    - {bag: [datacube.model.Dataset], geopolygon: query-poly, product_definitions: {product-name: datacube.model.Product}}
+# 5. box = self.input.group(bag)
+#    - {box: xarray.DataArray, geobox: None, load_natively: True, product_definitions: {product-name: datacube.model.Product}, geopolygon: query-poly}
+#    - Product.group(..., group_settings: dict) could be used to pass in 'output_crs', 'resolution', 'align'
+# Solution
+# 1. In .__init__.PolygonDrill.input_data, check if the bag datasets have a grid_spec and if not
+#    - get output_crs, resolution and/or align from datacube-wps-config.yaml
+#    - if output_crs is None, calculate the most common CRS from the bag datasets
+#    - call self.input.group(bag, output_crs, resolution, align)
 
```

Comment:
Suggest we delete this; this comment replaces it: https://github.com/csiro-easi/datacube-wps/issues/3#issuecomment-1322810398